client/src/constants/archiveParticipantsConstants.js

```diff
@@ -2,6 +2,11 @@ export const UnsuccessfulCohortReason = 'Unsuccessful cohort, does not wish to c
 
 export const PSIEducationUnderwayStatus = 'Post secondary education underway';
 
+export const ROSUnderwayStatus = 'Return of service underway';
+export const ROSCompleteStatus = 'Return of service complete';
+export const ROSCompletedType = { value: 'rosComplete', label: 'Return of service completed' };
+export const ROSReason = 'Completed all HCAP requirements';
```

Comment:
Can we get more specific with this naming? SuccessfulROSReason maybe? That's in-line with what we have for cohorts

Reply:
SuccessfulROSReason it is!